metadeploy/api/models.py

```diff
@@ -46,6 +45,7 @@
     preflight_failed,
     preflight_invalidated,
 )
+from .salesforce import _refresh_access_token
```

Comment:
This should be renamed to not use an initial underscore if it's going to be imported elsewhere.